lib/Conversion/LowerDIP/LowerDIPPass.cpp

```diff
@@ -44,6 +48,23 @@ using namespace mlir::arith;
 
 namespace {
 
+Value insertZeroConstantOp(MLIRContext *ctx, OpBuilder &builder, Location loc,
```

Comment:
Please move this function inside `Utils.h`. Also, please add a comment line explaining the motive of this function, similar to other functions in `Utils.h`.

Reply:
Done.